stixpy/product/scratch.py

```diff
@@ -14,12 +37,24 @@
 times = Time("2024-01-01T00:00:00") + np.arange(pd_shape[0]) * 2 * u.s
 energies = (1 + 2 * np.arange(pd_shape[-1])) * u.keV
 
-# Create a gwcs
-energy_coord = QuantityTableCoordinate(energies, names="energy", physical_types="em.energy")
+# Create table coords
 time_coord = TimeTableCoordinate(times, names="time", physical_types="time")
+energy_coord = QuantityTableCoordinate(energies, names="energy", physical_types="em.energy")
+
+# fake coords for detector and pixel
+detector_coord = QuantityTableCoordinate(
+    np.arange(pd_shape[1]) * u.dimensionless_unscaled, names="detector", physical_types="custom:detector"
+)
+pixel_coord = QuantityTableCoordinate(
+    np.arange(pd_shape[2]) * u.dimensionless_unscaled, names="pixel", physical_types="custom:pixel"
+)
+
+combine_coord = energy_coord & pixel_coord & detector_coord & time_coord
 
-combine_coord = time_coord & energy_coord
 
+# I'm not sure what should live in the meta vs extra_coords
 meta = Meta({"detector": ["a", "b", "c"]}, data_shape=pd_shape)
 
 scube = NDCube(data=pd, wcs=combine_coord.wcs, meta=meta)
+scube.extra_coords.add("integration_time", 0, [2, 3] * u.s, physical_types="time.duration")
```

Comment:
Following from the above comment, "integration time" should be considered metadata, not a coordinate.